maas/plugins/glance_api_local_check.py

```diff
@@ -16,55 +16,48 @@
 
 import argparse
 import collections
-
 import ipaddr
+import time
+
+from glanceclient import exc as exc
 from maas_common import get_auth_ref
-from maas_common import get_keystone_client
+from maas_common import get_glance_client
 from maas_common import metric
 from maas_common import metric_bool
 from maas_common import print_output
 from maas_common import status_err
 from maas_common import status_ok
-import requests
-from requests import exceptions as exc
+
 
 IMAGE_STATUSES = ['active', 'queued', 'killed']
 
 
 def check(auth_ref, args):
-    # We call get_keystone_client here as there is some logic within to get a
-    # new token if previous one is bad.
-    keystone = get_keystone_client(auth_ref)
-    auth_token = keystone.auth_token
-    api_endpoint = 'http://{ip}:9292/v1'.format(ip=args.ip)
-
-    s = requests.Session()
-
-    s.headers.update(
-        {'Content-type': 'application/json',
-         'x-auth-token': auth_token})
+    GLANCE_ENDPOINT = (
+        'http://{ip}:9292/v1'.format(ip=args.ip)
```

Comment:
The endpoints are hard coded everywhere so I did not change it, I fixed the more important stuff to use the glance client rather than then requests. We can roll this up in a later cycle and fix all the checks, by adding a port/protocol option